src/nat/llm/dynamo_llm.py

```
@@ -334,16 +433,173 @@ def create_httpx_client_with_dynamo_hooks(
+                    path,
+                    call_index,
+                    total_requests,
+                    osl,
+                    int(prediction.output_tokens.p90),
+                    iat,
+                    int(prediction.interarrival_ms.mean),
+                )
+            else:
+                logger.debug("No prediction found for path=%s, call_index=%d; using static values", path, call_index)
+
+        except Exception as e:
```

Ruff (0.14.14)

Do not catch blind exception: `Exception`

(BLE001)